protip.gemspec

```diff
@@ -14,7 +14,10 @@ Gem::Specification.new do |spec|
 
   spec.add_runtime_dependency 'activemodel', '>= 3.0.0', '< 5.0'
   spec.add_runtime_dependency 'activesupport', '>= 3.0.0', '< 5.0'
-  spec.add_runtime_dependency 'protobuf', '~> 3.5'
+
+  # Temporarily restricted. Protobuf bumped their dependency on multi_json, which is causing
+  # problems with compass.
+  spec.add_runtime_dependency 'protobuf', '~> 3.5.0'
```

Comment:
I believe this needs to be `<= 3.5.0`, the `~>` syntax will allow minor version increments